### PR TITLE
add new discourse dev, stage domain as origins to respective cdns

### PR DESCRIPTION
**Pull request by cmharlow**
Jira: https://mozilla-hub.atlassian.net/browse/SE-2162

What this PR does:
* adds new origins to dev, stage discourse CDN for new cluster apps
* (~possibly in future update after meeting with leo~): remove old origins for old cluster app, as okay with partial decomm

### CODE REVIEW

File: terraform/applications/discourse/cdn.tf [hunk 1]
```diff
@@ -12,6 +12,19 @@ resource "aws_cloudfront_distribution" "discourse" {
...
   origin {
     domain_name = local.workspace.discourse_cdn_zone
+    origin_id   = "discourse-pull-origin-old"
+    origin_path = ""
+
+    custom_origin_config {
+      http_port              = "80"
+      https_port             = "443"
+      origin_protocol_policy = "https-only" # Only talk to the origin over HTTPS
+      origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
```
Comment (bkochendorfer) [inline terraform/applications/discourse/cdn.tf:22]: We can probably trim this down to only include `TLSv1.2`. The origin here should prefer that and doesn't seem to support TLSv1 or TLSv1.1. https://www.ssllabs.com/ssltest/analyze.html?d=discourse.dev.mozit.cloud&s=52.26.54.173

**Review by bkochendorfer — COMMENTED** [review]
In the case here where we have multiple `origin`s is the preference ranked? As in it will send all requests to the `old` id for now?

**Comment by cmharlow** [comment]
> In the case here where we have multiple `origin`s is the preference ranked? As in it will send all requests to the `old` id for now?

brett, there is 1 origin now migration is done.

**Review by duallain — APPROVED** [review]
LGTM, discourse.stage|prod.mozit.cloud look like valid orgins to me, so should be safe to point the cdn at them

**Pull request merged**